Sources/Components/TextField/TextField.swift

```diff
@@ -297,6 +303,10 @@ public class TextField: UIView {
         return evaluate("[A-Z0-9a-z._%+-]+@[A-Za-z0-9.-]+\\.[A-Za-z]{2,64}", with: emailAdress)
     }
 
+    fileprivate func isValidPhoneNumber(_ phoneNumber: String) -> Bool {
+        return evaluate("^[0-9]{8,11}$", with: phoneNumber)
```

Comment:
Not sure how users might input their phonenumbers, but this doesn't seem to validate numbers prefixed with country code or with spacing in between.

Like:
- `+47xxxxxxxx`  ← 11 characters, but `+` isn't allowed according to regex.
- `0047xxxxxxxx` ← 12 characters, all digits.
- `xxx xx xxx`   ← The "usual" way of formatting mobile numbers.
- `xx xx xx xx`  ← The "usual" way of formatting landline numbers.

Reply:
Good observation, maybe this should be configurable too? Because I guess different backends might want this sent in different ways, maybe that we do the conversion to that format after, but if backend doesn't support `+` we should perhaps not allowed it as input either 🤔 

We should maybe ask what is allowed in this first use case.

Reply:
I took `[0-9]{8,11}` from the webform this view is based on. But I think it's a good idea to make it configurable.